libraries/SPI/src/utility/xmc_spi_conf.h

```diff
@@ -60,10 +60,6 @@ extern XMC_SPI_t XMC_SPI_0;
 #define NUM_SPI  			1
 extern XMC_SPI_t XMC_SPI_0;
 
-#elif defined(XMC1400_Arduino_Kit)
```

Comment:
Same as [xmc_spi_conf.c](https://github.com/Infineon/XMC-for-Arduino/pull/195/files#diff-f0a5b6e4a58530712f1cbce9463eed9daedae46d0f0acb1f8fde3641bd66ed4c)